### PR TITLE
Feat/f2p 66 new branding regulator

### PR DESCRIPTION
**Pull request by dc-andysign**
### What

For story - [F2P-66](https://digicatapult.atlassian.net/browse/F2P-66)

As per what has been suggested, changed the name of the last Actor to Regulator.

> **_Regulator - Safe2Fly_**

> _Safe2Fly are in charge of issuing and regulating all flyer IDs, as well as regulating all issuers. Ultimately, it’s their responsibility to ensure that all drone flying is carried out in a legal and safe manner. They also have the ability to revoke any flyer IDs if they wished._ 

> _Safe2Fly, on receipt of a passed training certificate from a holder, will issue a licence in response. They also have the ability to display all the licences for a given holder’s flyer ID._

### Validation

- [x] - replace text everywhere

### CODE REVIEW

File: docker/docker-compose.yaml [hunk 1]
```diff
@@ -65,29 +65,29 @@ services:
...
       - ACAPY_AUTO_VERIFY_PRESENTATION=true
     ports:
       - 8051:8002
     networks:
       - von
-  react-authority:
-    image: ghcr.io/digicatapult/veritable-authority:latest
-    container_name: veritable-react-authority
+  react-regulator:
+    image: ghcr.io/digicatapult/veritable-regulator:latest
```
Comment (mattdean-digicatapult) [inline docker/docker-compose.yaml]: this repo doesn't exist at the moment so you'll need to do a corresponding PR on the authority repo to start with. Then I'll do the repo rename

Comment (dc-andysign) [reply docker/docker-compose.yaml]: Oh you re back.. ok, back to `ghcr.io/digicatapult/veritable-authority:latest` for now.

File: docker/docker-compose.yaml [hunk 2]
```diff
@@ -65,29 +65,29 @@ services:
...
       - ACAPY_AUTO_VERIFY_PRESENTATION=true
     ports:
       - 8051:8002
     networks:
       - von
-  react-authority:
+  react-regulator:
     image: ghcr.io/digicatapult/veritable-authority:latest
-    container_name: veritable-react-authority
+    container_name: veritable-react-regulator
```
Comment (n3op2) [inline docker/docker-compose.yaml:124]: why. the container name is different, did I miss something?

Comment (n3op2) [reply docker/docker-compose.yaml:124]: https://docs.google.com/document/d/1K2WAIm59QSZcKnPQ-22Gb05IA6Hb88JT5vU6PTBUu0E/edit# sorry dont have access to this document

Comment (dc-andysign) [reply docker/docker-compose.yaml:124]: Check Apollo

**Review by n3op2 — DISMISSED** [review]

**Review by mattdean-digicatapult — DISMISSED** [review]

**Review by jonmattgray — DISMISSED** [review]

**Review by jonmattgray — APPROVED** [review]

**Pull request merged**